benchbuild/projects/benchbuild/xz.py

```diff
@@ -31,30 +31,32 @@ def compile(self):
         clang = compiler.cc(self)
         with local.cwd(unpack_dir):
             configure = local["./configure"]
+            configure = run.watch(configure)
             with local.env(CC=str(clang)):
-                run.run(configure["--enable-threads=no", "--with-gnu-ld=yes",
-                                  "--disable-shared",
-                                  "--disable-dependency-tracking",
-                                  "--disable-xzdec", "--disable-lzmadec",
-                                  "--disable-lzmainfo", "--disable-lzma-links",
-                                  "--disable-scripts", "--disable-doc"])
+                configure("--enable-threads=no", "--with-gnu-ld=yes",
+                          "--disable-shared", "--disable-dependency-tracking",
+                          "--disable-xzdec", "--disable-lzmadec",
+                          "--disable-lzmainfo", "--disable-lzma-links",
+                          "--disable-scripts", "--disable-doc")
 
-            run.run(make["CC=" + str(clang), "clean", "all"])
+            make_ = run.watch(make)
+            make_("CC=" + str(clang), "clean", "all")
 
-    def run_tests(self, runner):
+    def run_tests(self):
         unpack_dir = local.path('xz-{0}'.format(self.version))
         _xz = wrapping.wrap(unpack_dir / "src" / "xz" / "xz", self)
+        _xz = run.watch(_xz)
```

Comment:
Hmm here you name the variable `_name` before it was `name_`

Reply:
Fixed in cc611b4a00e299b1eed8084f79553c9b3259e800